styles.css

```diff
@@ -1 +1,60 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/*Maak hier jouw CSS!*/
+.container {
+    display: grid;
+    background-color: hsla(205, 100%, 90%, 1);
+    row-gap: 16px;
+    column-gap: 16px;
+    grid-template-columns: 1fr;
+    grid-template-rows: 50px auto 50px 50px 100px 50px 50px;
+    grid-template-areas:
+        "item-a"
+        "item-b"
+        "item-c"
+        "item-d"
+        "item-e"
+        "."
+        "item-f"
+;
+}
+
+.item {
+    background-color: hsla(205, 100%, 40%, 1);
+    color: hsla(360, 100%, 100%, 1);
+}
+
+#item-a {
+    grid-area: item-a;
+}
+#item-b {
+    grid-area: item-b;
+}
+#item-c {
+    grid-area: item-c;
+}
+#item-d {
+    grid-area: item-d;
+}
+#item-e {
+    grid-area: item-e;
+}
+#item-f {
+    grid-area: item-f;
+}
+@media screen and (min-width: 576px) {
```

Comment:
Netjes gedaan. Ik heb het er zelf een beetje brute-force ingeramd. Maar deze oplossing is zoals het moest. Top!